docs/2____common_concepts.adoc

```diff
@@ -19,32 +19,25 @@ To uniquely map between the FMU variables and the network variables, FMU variabl
 [#figure-terminal-relations]
 image::terminal_relations.svg[width=60%, align="center"]
 
-#TODO AJ#: explanation of clock and binary (check names!) variables for low cut; clock type influences and flexibilities; not use of nextEventTime
-
 All variables have structured names that indicate how these variables are related and to which abstraction layer they belong.
 For the high cut, variables represent signals and their clocks.
-For the low cut, variables represent frames and their clocks.
-Input and output variables show if these variables belong to signals or frames that are received or sent, respectively.
+For the low cut, variables represent send and receive bus traffic and their respective clocks.
+Input and output variables show if these variables belong to signals or bus traffic that are received or sent, respectively.
 Input clocks must be triggered and are connected by the importer to their corresponding output clocks of potentially different clock types.
 For example, a periodic output clock indicates that the FMU can only send signals or frames periodically, while a countdown clock indicates that the FMU can send signals at varying time steps.
 
 Using output clocks of different types allows simulations with different timing accuracy and performance.
-These clocks allow the FMU to specify the next send event efficiently and accurately, the importer will trigger these clocks according to these clock specifications as accurately as it can.
+Some clock types allow the FMU to specify the next send event efficiently and accurately, the importer will trigger these clocks according to these clock specifications as accurately as it can.
```

Comment:
We need a big hint "You can use trigged Clocks but they may be not your friend". Maybe at another place within the document or using a link and write a chapter into the implementer's guide? See also Triggered clocks: forbidding or discouraging? #14

Reply:
Added two sentences to that effect in 2__common_concepts. Please check.